<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smart</class>
 <widget class="QWidget" name="smart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>smart</string>
  </property>
  <widget class="QPushButton" name="pb1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/image/case1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>755</width>
     <height>430</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="l_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>这是风扇界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_fan">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>561</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_fan">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_fan">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="first">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>200</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>一档</string>
     </property>
    </widget>
    <widget class="QPushButton" name="second">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>280</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>二档</string>
     </property>
    </widget>
    <widget class="QPushButton" name="third">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>350</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>三档</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="l_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>卧室的小米空调</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>35</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>室外温度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>35</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>空气湿度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>180</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>230</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>风速</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>230</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>风向</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>230</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>扫风</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>280</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 35pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>280</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 36pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>330</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>睡眠</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>160</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>风速：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>160</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>风向</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>583</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>手动风</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>70</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>45</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>45</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>145</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>145</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>101</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>225</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>实时温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>实时湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>25℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>24.6℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>32%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>31.7%</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>这是查看pm2.5界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>170</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;隶书&quot;;</string>
     </property>
     <property name="text">
      <string>PM2.5浓度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>250</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>这是汽车界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>361</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="lan1">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>260</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>亮起灯光</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>171</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="lan2">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>329</y>
       <width>80</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭灯光</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>这是厨房界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>551</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>火焰传感器</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>这是卧室界面</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>721</width>
       <height>371</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>这是说明书</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;物联网智能风扇可通过手机等设备远程控制，能根据环境温&lt;/p&gt;&lt;p&gt;度自动调节风速，还可设置定时，使用方便且节能。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;物联网智能空调可远程操控，用户能在到家前提前开启。它还&lt;/p&gt;&lt;p&gt;能根据环境自动调节温度，节能且提供舒适体验。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;物联网智能汽车具备众多令人惊叹的功能。它能通过车联网实&lt;/p&gt;&lt;p&gt;现实时交通信息获取，自动规划最优路线，躲避拥堵。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; 物联网智能温湿度检测器可实时监测环境温湿度，数据远&lt;/p&gt;&lt;p&gt;程传输。能预警异常，&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_9">
    <widget class="QLabel" name="l_gif">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>451</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;妈妈：记得买菜。&lt;br/&gt;爸爸：记得喝水。&lt;br/&gt;我：好好学习。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;楷体&quot;;
</string>
     </property>
     <property name="text">
      <string>家庭备忘录</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>248</width>
       <height>156</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_page1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>风扇</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>335</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>留言版</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>空调</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>155</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>pm2.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>汽车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>245</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>厨房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>卧室</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>温湿度</string>
   </property>
  </widget>
  <widget class="textScroll" name="scroll3" native="true">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>430</y>
     <width>741</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>4</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_page8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>80</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>说明书</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>textScroll</class>
   <extends>QWidget</extends>
   <header>textscroll.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
